service/health/searcher/types.go

```diff
@@ -72,3 +74,20 @@ func NewNodesStatus(cluster clusterInfo) []NodeStatus {
 	}
 	return result
 }
+
+func FindInAddresses(addresses []v1.NodeAddress, address_type v1.NodeAddressType) string {
```

Comment:
None of these functions need to be public IMO

Reply:
Learned about meaningful function name case today :+1:

Reply:
Another day another learning :D